helper/resource/importstate/import_block_as_first_step_test.go

```diff
@@ -41,7 +41,13 @@ func TestImportBlock_AsFirstStep(t *testing.T) {
 				Config: `resource "examplecloud_container" "test" {
 					name = "somevalue"
 					location = "westeurope"
-				}`,
+				}
+
+				import {
```

Comment:
This test change is a consequence of #484.

Curiously, the tests did not run in https://github.com/hashicorp/terraform-plugin-testing/pull/484 💭